Compiler/COMMON/sgl.linker

```diff
@@ -1,4 +1,3 @@
-OUTPUT_FORMAT(coff-sh)
```

Comment:
@abrasive I had to revert this block of code (OUTPUT_FORMAT) otherwise the game no longer worked in some cases. Unfortunately, I don't have time to watch at the moment